.changeset/slick-seas-learn.md

```diff
@@ -0,0 +1,20 @@
+---
+'@directus/api': major
+---
+
+Fixed snapshots including untracked collections
+
+::: notice
```

Comment:
@licitdev updated the changeset to major, let me know if the wording is acceptable.